packages/mobile/locales/en-US/nuxNamePin1.json

```diff
@@ -26,6 +29,7 @@
   "fullName": "Full Name",
   "importIt": "Restore Existing Wallet",
   "cancel": "Cancel",
+  "accept": "Accept",
```

Comment:
consider putting single-word strings like this in global.json instead so we can use it throughout the app